modules/addons/ratelimit/main.tf

```diff
@@ -0,0 +1,37 @@
+provider "helm" {
+  kubernetes {
+    host                   = var.k8s_host
+    cluster_ca_certificate = base64decode(var.k8s_cluster_ca_certificate)
+    token                  = var.k8s_client_token
+  }
+}
+
+provider "kubectl" {
+  host                   = var.k8s_host
+  cluster_ca_certificate = base64decode(var.k8s_cluster_ca_certificate)
+  token                  = var.k8s_client_token
+  load_config_file       = false
+}
+
+resource "random_password" "redis" {
```

Comment:
let's use "tsb_password" as a variable to set set password for all the related tools, for example argocd password is using tsb_password as well.

Reply:
ignore my previous comment on redis, agree to have a random password set for backend `redis`. It only makes sense to have a user facing tools password to be set to `tsb_password`